web/src/components/DandiFooter.vue

```diff
@@ -12,6 +12,15 @@ export default {
     version() {
       return process.env.VUE_APP_VERSION;
     },
+
+    githubLink() {
+      const gitRev = process.env.VUE_APP_GIT_REVISION;
+      if (gitRev) {
+        return `https://github.com/dandi/dandiarchive/commit/${gitRev}`;
+      } else {
+        return "https://github.com/dandi/dandiarchive";
+      }
```

Comment:
duplication allergy in me is really unhealthy. Isn't there some syntax support for smth like

```suggestion
      return `https://github.com/dandi/dandiarchive` + (${gitRev} ? `/commit/${gitRev}` : ``);
```

but totally feel to ignore my allergic reaction here -- it is a url/code which is not to change etc

Reply:
Even better would be something like

```javascript
      return `https://github.com/dandi/dandiarchive${gitRev ? `/commit/${gitRev}` : ""}`;
```

but despite your allergy to duplication (which I share a mild version of, truth be told), I find our condensed versions far less legible than what I have here with the separated `return` statements.